php_phongo.c

```diff
@@ -2583,6 +2583,16 @@ PHP_MINFO_FUNCTION(mongodb)
 #else
 	/* Bundled libraries, buildtime = runtime */
 	php_info_print_table_row(2, "libmongoc version", MONGOC_VERSION_S);
+# if HAVE_MONGOC_SSL
```

Comment:
These constants were not defined in the .h files that I needed it to be in:

```
src/libmongoc/src/mongoc/mongoc-config.h:#define MONGOC_ENABLE_SSL 1
win32/mongoc-config.h:#define MONGOC_ENABLE_SSL 1
```

And I don't want to expose vendored library's defines outside of it. Having specific defines for them makes for a cleaner separation.
```